src/main/java/models/Player.java

```diff
@@ -3,12 +3,22 @@
 /**
  * Created by krisna on 3/11/2016.
  */
-public class Player extends Game
-{
+public class Player extends Game {
     @Override
-    public void deal(int index)
-    {
-        hands.get(index).add(deck.get(deck.size()-1));
-        deck.remove(deck.size()-1);
+    public void deal(int index) {
+        hands.get(index).add(deck.get(deck.size() - 1));
+        deck.remove(deck.size() - 1);
     }
-}
+
```

Comment:
The win lose and stay functionality are more complicated than that. 
I have build a little bit of the stay functionality, I believe if we combine it, it will be perfect.